backoffice/src/Services/WreckService.cs

```diff
@@ -42,7 +42,7 @@ public async Task<Wreck> SaveWreckAsync(Wreck wreck)
     public async Task<Wreck> GetWreckAsync(Guid id) =>
         await _repo.GetWreckAsync(id);
 
-    public async Task<Guid> GetWreckIdAsync(WreckForm wreckForm)
+    public async Task<Guid> SaveWreckFormAsync(WreckForm wreckForm)
```

Comment:
I wonder if this should be separated out into two methods (for following single responsibility principle). Seems OTT for the size of the methods but could improve readability.

Reply:
I generally like having a single function -> Upsert

That then works out whether it should Update or Insert (Add)

Then calls either Update or Add

Reply:
Then Update/Add are private and never called directly externally